.vscode/settings.json

```diff
@@ -1,4 +1,9 @@
 {
     "editor.formatOnSave": true,
-    "editor.tabSize": 2
+    "editor.tabSize": 2,
+    "workbench.colorCustomizations": {
```

Comment:
Undo this change.

Try to never have any unnecessary changes in a PR, and only the changes needed.